app/services/activity/updater.rb

```diff
@@ -39,7 +39,7 @@ def set_parent
 
       case activity.level.to_sym
       when :programme then UpdateActivityAsProgramme.new(activity: activity, parent_id: parent_id).call
-      when :project then UpdateActivityAsProject.new(activity: activity, parent_id: parent_id).call
+        # when :project then UpdateActivityAsProject.new(activity: activity, parent_id: parent_id).call
```

Comment:
This service is accumulating commented lines. Could you remove them at the point where they become deprecated?

Reply:
Good point - think I made the mistake of commenting them out whilst I was figuring this all out, but didn't go back and removing them entirely 👍